Makefile

```diff
@@ -37,7 +37,7 @@ docker-build:
 			. \
 			|| exit 1; \
 	done
-	sudo rm -rf vendor
+	rm -rf vendor
```

Comment:
Wondering why we had added sudo. and do you see any issues with sudo ?

Reply:
not sure when we added it. but when we try to run on the laptop with sudo it asks for password. so thought better to remove it. not needed in any case.